src/ui/public/errors.js

```diff
@@ -165,11 +165,15 @@ export class DuplicateField extends KbnError {
  * A saved object was not found
  */
 export class SavedObjectNotFound extends KbnError {
-  constructor(type, id) {
+  constructor(type, id, additionalMessage) {
     const idMsg = id ? ` (id: ${id})` : '';
-    super(
-      `Could not locate that ${type}${idMsg}`,
-      SavedObjectNotFound);
+    let message = `Could not locate that ${type}${idMsg}`;
```

Comment:
Using html directly here doesn't appear to give us the desired result:

<img width="1649" alt="screen shot 2017-07-10 at 11 34 40 am" src="https://user-images.githubusercontent.com/627123/28026336-b570e0c4-6563-11e7-8b18-ea1e283be1b7.png">